spec/features/rooms/index_spec.rb

```diff
@@ -0,0 +1,29 @@
+
+require 'rails_helper'
+
+RSpec.describe 'Rooms Index', type: :feature do
+  let(:hotel1) { Hotel.create(name: 'Echo Mountain Inn', location: 'Echo Mountain') }
+  let(:hotel2) { Hotel.create(name: 'Alpine Resort', location: 'Alpine Valley') }
+  let(:room1) { Room.create(rate: 125, suite: 'Presidential', hotel: hotel1) }
+  let(:room2) { Room.create(rate: 200, suite: 'Deluxe', hotel: hotel2) }
+  let(:guest1) { Guest.create(name: 'Charlize Theron', nights: 3) }
+  let(:guest2) { Guest.create(name: 'Tom Hanks', nights: 2) }
+
+  before do
+    GuestRoom.create(guest: guest1, room: room1)
+    GuestRoom.create(guest: guest2, room: room2)
+  end
+
+  it 'displays a list of all rooms with their details and guest count' do
+    visit "/rooms"
+    expect(page).to have_content(room1.suite)
+    expect(page).to have_content(room1.rate)
+    expect(page).to have_content(hotel1.name)
+    expect(page).to have_content(room1.guests.count)
+
+    expect(page).to have_content(room2.suite)
```

Comment:
The test setup is fine, but I find it more useful to test using strings exactly as it will appear on the page. Otherwise, capybara will look _**anywhere**_ in the page document for each thing separately, regardless of the order in which the tests appear. Example: 
```
expect(page).to have_content("Stayed at Alpine Resort's Deluxe suite for $200/night 2 times")
```
this is a more accurate test because we're expecting all the data being asked for is on the page in this specific context. The test you currently have is asking "Hey capybara, is the number `2` present anywhere on the page?", which will pass if there's a timestamp on the page and it's 12:00. So, accurate feature tests do matter in the long run.